openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/KPILatestResultsV1.tsx

```diff
@@ -37,7 +37,11 @@ const KPILatestResultsV1: FC<Props> = ({ kpiLatestResultsRecord }) => {
   }, [kpiLatestResultsRecord]);
 
   return (
-    <Space className="w-full p-t-lg p-r-xs" direction="vertical" size={48}>
+    <Space
+      className="w-full p-t-lg p-r-xs"
+      data-testid="kpi-latest-result-container"
+      direction="vertical"
+      size={48}>
```

Comment:
Is this size intentional?

Reply:
Yes, it's intentional, to have a gap between 2 KPI
<img width="238" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/71748675/f252852d-d899-4c23-a652-74f24ece67ad">